demo-app/cypress/integration/counter.spec.js

```diff
@@ -0,0 +1,18 @@
+describe('Counter Component', () => {
+    it('increments and decrements the counter', () => {
+      // Visit the page with the Counter component
+      cy.visit('http://localhost:3000/'); // Update the URL as needed
+  
+      // Find the Counter Value element and assert its initial value
+      cy.get('h1').should('have.text', 'Counter Value: 0');
```

Comment:
```suggestion
      cy.get('h1').should('have.text', 'Counter Value: 10');
```
Because you render this component with initial value 10